sumpy/kernel.py

```diff
@@ -263,18 +253,6 @@ def get_derivative_transformation_at_source(self, expr_dict):
         """
         return expr_dict
 
-    def get_derivative_transformation_at_target(self, expr_dict):
```

Comment:
What's happening here? Do you intend for this to remain part of the `Kernel` interface? If so, I think it'd be good to leave an `@abstractmethod` behind. In that case, it should probably also be roped into the class documentation.